pmlc/dialect/xsmm/ir/ops.cc

```diff
@@ -21,29 +21,29 @@ void XSMMDialect::initialize() {
 }
 
 //
-// ---- GemmInvokeOp ----
+// ---- SGemmInvokeOp ----
```

Comment:
What is SGemm here?  Also, maybe we should put renames (this/indicies/etc) in as separate NFC changes?

Reply:
Sure. It's following the naming convention of xsmm (s = single, d = double, etc)

Reply:
Actually, this rename is important, because it includes the data type, which is actually one of the main parts of this PR.